modules/network-observability-flowcollector-kafka-config.adoc

```diff
@@ -20,15 +20,14 @@ You can configure the `FlowCollector` resource to use Kafka for high-throughput
 . Modify the `FlowCollector` resource for {product-title} Network Observability Operator to use Kafka, as shown in the following sample YAML:
 
 .Sample Kafka configuration in `FlowCollector` resource
-[id="network-observability-flowcollector-configuring-kafka-sample_{context}"]
 [source, yaml]
 ----
-apiVersion: flows.netobserv.io/v1beta1
+apiVersion: flows.netobserv.io/v1beta2
 kind: FlowCollector
 metadata:
   name: cluster
 spec:
-  deploymentModel: KAFKA                                    <1>
+  deploymentModel: Kafka                                    <1>
```

Comment:
I can't suggest update from here directly since it's not updated code but you sould also update the <1> mention below:
```
<1> Set `spec.deploymentModel` to `Kafka` instead of `Direct` to enable the Kafka deployment model.
```

Reply:
Got it, thanks for pointing this one out as well.